WcaOnRails/app/webpacker/stylesheets/static_pages/logo.scss

```diff
@@ -0,0 +1,7 @@
+.wca-logo-information {
+  div {
+    /* Apply same styles as to 'p' tag, as tag is not allowed in context */
+    line-height: 1.4285em;
+    margin: 0 0 1em;
+  }
+}
```

Comment:
I'm not a big friend of "magic numbers" anywhere in the code, particularly not in CSS.

What about wrapping the `div` returned by `I18nHtmlTranslate` inside a `p` tag to get this styling "for free"? Something along the lines of
```jsx
<p>
  <I18nHtmlTranslate ...foo... />
</p>
```

Reply:
I did try this, but it throws an error in the console at runtime (which I assume is worse) because `div`s are not allowed inside `p` tags. The alternative I think, which would make more sense in general, would be putting the contents of the `I18nHtmlTranslate` in a `span` rather than a `div` as you would expect to use it in a `p` most of the time.

Reply:
Yes, having a span totally makes sense! Would you mind submitting a separate PR that
1. Changes the `I18nHtmlTranslate` component to use `<span>` instead of `<div>`
2. Adds a `// eslint-disable-next-line react/no-danger` comment in that file to prevent those annoying CI warnings?

Reply:
Yep I'll get on that

Reply:
Awesome, thanks a lot!